ui/timeline/src/commonMain/kotlin/social/androiddev/timeline/FeedCard.kt

```diff
@@ -0,0 +1,217 @@
+package social.androiddev.timeline
+
+//import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import social.androiddev.common.composables.UserAvatar
+import social.androiddev.common.theme.MastodonTheme
+
+/**
+ * Stateful
+ */
+@Composable
+fun FeedCard(
+    state: FeedItemState,
+    contentPadding: PaddingValues = PaddingValues(8.dp),
+    modifier: Modifier = Modifier,
```

Comment:
I believe the `modifier` should start before `contentPadding`.  

According to [Modifier docs](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier):

> Composables that accept a [Modifier](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier) as a parameter to be applied to the whole component represented by the composable function should name the parameter modifier and assign the parameter a default value of [Modifier](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier). It should appear as the first optional parameter in the parameter list; after all required parameters (except for trailing lambda parameters) but before any other parameters with default values. Any default modifiers desired by a composable function should come after the modifier parameter's value in the composable function's implementation, keeping [Modifier](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier) as the default parameter value

Reply:
Yup - good catch. I thought I did this in all of them and must have missed it here. Maybe we can set up some rules to catch this as well.

Reply:
The Detekt setup I am adding has all these rules. You can also check out [these rules](https://twitter.github.io/compose-rules/rules) compiled by Twitter